fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -147,6 +147,7 @@ private void auditAfterExec(String origStmt, StatementBase parsedStmt, PQuerySta
                 .setState(ctx.getState().toString()).setErrorCode(ctx.getErrorCode()).setQueryTime(elapseMs)
                 .setScanBytes(statistics == null ? 0 : statistics.scanBytes)
                 .setScanRows(statistics == null ? 0 : statistics.scanRows)
+                .setCpuCostNs(statistics == null || statistics.cpuCostNs == null ? 0 : statistics.cpuCostNs)
```

Comment:
may be we need add set setCpuCostNs to the last.otherwise some auditlog parsing tools couldn't work

Reply:
Actually it is just a builder, and has not effect with the ordering.